main.py

```diff
@@ -288,6 +309,26 @@ async def loop(adb_instance: ADB, config: AluneConfig):
                     screenshot = await adb_instance.get_screen()
                     search_result = screen.get_button_on_screen(screenshot, Button.exit_now)
                     game_state = await get_game_state(screenshot)
+
+                    # If the surrend feature is enabled, we expand the top bar to check if
+                    # we're in a surrend-possible phase
+                    if config.get_auto_surrend():
+                        await adb_instance.click_button(Button.expand_top_bar)
+                        await asyncio.sleep(1)
+                        is_phase_3_2 = screen.get_on_screen(screenshot, Image.PHASE_3_2_FULL)
+                        if is_phase_3_2:
+                            if _random_delay == 0:
+                                logger.info("We're in a phase we can surrend ! No delay have been set in config file.")
+                                logger.info("Surrendering now !")
+                                surrend_game(adb_instance)
+                            else:
+                                logger.info(
+                                    "We're in a phase we can surrend. A random delay have been set in config file."
+                                )
+                                logger.info("Surrendering in {_random_delay} second(s) !")
+                                surrend_timer = Timer(_random_delay, await surrend_game(adb_instance))
+                                surrend_timer.start()
```

Comment:
This could be extracted into a function to increase readability.

It doesn't _need to_, but it could return true/false based on if it needs to break from the game loop (the break below this)